changelog/pending/20240305--sdk-nodejs--adds-support-to-suppress-progress-and-suppress-outputs-parameters.yaml

```
@@ -0,0 +1,4 @@
+changes:
+- type: feat
+  scope: sdk/nodejs
```

```suggestion
  scope: auto/go,nodejs,python
```

I just realized what you meant about the changelog. You were right of course!